tailwind.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  purge: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
+  darkMode: false, // or 'media' or 'class'
+  theme: {
+    extend: {},
+  },
+  variants: {
+    extend: {},
+  },
+  plugins: [],
+};
```

Comment:
- [ ] I got this issue when I tried to compile your file using `npm start`. Please fix it.
![Screenshot from 2022-02-23 16-16-33](https://user-images.githubusercontent.com/63087491/155349275-23252d7a-9406-4b85-ba64-528f71eda5b8.png)